### ISSUE TITLE
Alt Text Incorrect for Table View and Cases Buttons

### ISSUE THREAD

**Issue by aarontitus** [CrisisCleanup/crisiscleanup-3-web#714]
**Describe the bug**
Layers, Filters, Download, "...", Unclaim, and Update Status buttons alt text are all "Button" despite crisiscleanup-3-web\src\pages\Cases.vue correctly listing :alt properly

**To Reproduce**
1. Go to "Work"
2. Click on Table View
3. Hover over any of the buttons in the screenshot. Bug: All of the alt text is "Button."

**Expected behavior**
Alt text should be as defined in crisiscleanup-3-web\src\pages\Cases.vue.

**Screenshots**
![Alt Text Issue](https://user-images.githubusercontent.com/4184619/83103131-5969a380-a073-11ea-882e-484b85bb8a60.png)

**Desktop (please complete the following information):**
 - Win 10, FF 76
